wgpu/src/lib.rs

```diff
@@ -1647,6 +1647,22 @@ impl Instance {
             .instance_as_hal::<A, F, R>(hal_instance_callback)
     }
 
+    /// Create an new instance of wgpu from a wgpu-core instance.
+    ///
+    /// # Arguments
+    ///
+    /// - `core_instance` - wgpu-core instance.
+    ///
+    /// # Safety
+    ///
+    /// Refer to the creation of wgpu-core Instance.
+    #[cfg(any(not(target_arch = "wasm32"), feature = "webgl2"))]
```

Comment:
This ought to match the condition used to select `backend/direct.rs`, shouldn't it? That's `"webgl"`,not `"webgl2"`.